Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'rubocop', '~>0.81.0', 'tty-box'
```

Comment:
- When you add gems to the `Gemfile` you **don't add it with a comma.** Although it seems that is the best way to do it, Ruby doesn't work like that. You need to **add new gems in their respective line.**

**For example**
```
  gem 'rubocop'
  gem 'tty-box'
  # gem 'whatever-gem'
```

- [ ] Separate new gems in a new line

Reply:
Thanks for your comment. It helps to understand.